src/components/IstioWizards/IstioWizardActions.ts

```diff
@@ -82,6 +82,10 @@ const SERVICE_UNAVAILABLE = 503;
 
 export const KIALI_WIZARD_LABEL = 'kiali_wizard';
 
+export const fqdnServiceName = (serviceName: string, namespace: string): string => {
+  return serviceName + '.' + namespace + serverConfig.istioIdentityDomain;
```

Comment:
Am I having a configuration issue or there is one `.` missing between the namespace and the domain?

Reply:
Ouch ! you are absolutely right, my bad !!